client-jooq/src/main/kotlin/app/cash/backfila/client/jooq/JooqBackfill.kt

```diff
@@ -85,6 +85,12 @@ abstract class JooqBackfill<K, Param : Any> : Backfill {
    */
   abstract fun backfill(backfillBatch: BackfillBatch<K, Param>)
 
+  /**
+   * Hook that gives you a place to finalize the backfill.
+   * This is called when the backfill is complete or errored, errors are ignored
```

Comment:
Errors should not be ignored. If a customer wants to ignore errors they can do that themselves.